src/components/ClientCard/ClientCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { setActiveChat } from '../../store/Chat/actions';
+import './ClientCard.css';
+
+const ClientCard = props => (
+  <li className="ClientCard" >
+    <button onClick={() => props.setActiveChat(props.chatId)}>
```

Comment:
do you mean that the `<Button>` component should be styleable to not be a button but wrap anything that needs an onclick?

- if so, I think that's a valuable idea -- you would probably just want to make it receive a prop for any classNames so that it could be passed down to whatever is actually being styled... so I guess that would make it a higher order button component (probably misusing that term) But basically, anything that needs an onclick, just gets wrapped in a `<Button type="wrapper">` so as not to apply any of the unique styling of our button commponent when used elsewhere.

hope that made sense. 

we could also just custom style any button like this situation as they come.

I also am not against setting it to a "warning" and dealing with it later.